var/spack/repos/builtin/packages/aom/package.py

```diff
@@ -0,0 +1,19 @@
+from spack import *
+
+class Aom(CMakePackage):
```

Comment:
```
var/spack/repos/builtin/packages/aom/package.py:3: [E302] expected 2 blank lines, found 1
var/spack/repos/builtin/packages/aom/package.py:16: [E225] missing whitespace around operator
var/spack/repos/builtin/packages/aom/package.py:19: [W391] blank line at end of file
```
Also, this package is missing a license.

Reply:
Thanks Adam and apologies for the simple mistakes.
License added and silly syntax mistakes hopefully cleaned up.